src/System.Net.Http/src/System/Net/Http/Managed/HttpProxyConnectionHandler.cs

```diff
@@ -105,8 +106,12 @@ protected internal override Task<HttpResponseMessage> SendAsync(HttpRequestMessa
 
                             if (await AuthenticationHelper.TrySetDigestAuthToken(request, credential, digestResponse, HttpKnownHeaderNames.ProxyAuthorization).ConfigureAwait(false))
                             {
+
+                                // TBD: We should drain response before disposing so it goes back to connection pool.
```

Comment:
This should be tagged with the issue number #23156.  We also generally use "TODO" rather than "TBD".

Reply:
fixed.